diff --git a/configure" "b/configure
index 05da189..26390d3 100644
--- a/configure
+++ b/configure
@@ -3339,10 +3339,10 @@ printf "%s\n" "continue configure in default builddir \"./$ax_enable_builddir\""
       srcdir=`echo "$ax_enable_builddir" |
               sed -e 's,^\./,,;s,[^/]$,&/,;s,[^/]*/,../,g;s,[/]$,,;'`
       # going to restart from subdirectory location
-      test -f $srcdir/config.log   && mv $srcdir/config.log   .
-      test -f $srcdir/confdefs.h   && mv $srcdir/confdefs.h   .
-      test -f $srcdir/conftest.log && mv $srcdir/conftest.log .
-      test -f $srcdir/$cache_file  && mv $srcdir/$cache_file  .
+      test -f $srcdir/config.log   && cp $srcdir/config.log   .
+      test -f $srcdir/confdefs.h   && cp $srcdir/confdefs.h   .
+      test -f $srcdir/conftest.log && cp $srcdir/conftest.log .
+      test -f $srcdir/$cache_file  && cp $srcdir/$cache_file  .
       { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: ....exec $SHELL $srcdir/$0 \"--srcdir=$srcdir\" \"--enable-builddir=$ax_enable_builddir\" ${1+\"$@\"}" >&5
 printf "%s\n" "....exec $SHELL $srcdir/$0 \"--srcdir=$srcdir\" \"--enable-builddir=$ax_enable_builddir\" ${1+\"$@\"}" >&6; }
       case "$0" in # restart
